### PR TITLE
Added basic tests for each api method of each service. These tests do…

### PR DESCRIPTION
**Pull request by zmoore-usgs**
… not cover all edge cases at this time, but are just minimal functionality tests.

### CODE REVIEW

**Comment by isuftin** [comment]
You probably want to update https://github.com/USGS-CIDA/MLR-Integration-Testing/blob/master/jmeter-docker/base/Dockerfile#L6 to be 5.0 or specify the incoming ARG on build? Probably better to specify 5.0 in the Dockerfile

**Comment by isuftin** [comment]
I'm going to make a PR to your branch in a few...

**Comment by zmoore-usgs** [comment]
Okay, I could also make those changes easily enough

**Comment by isuftin** [comment]
@zmoore-usgs The issue on my end is that I'm testing through the scripting/tests and making changes locally to do so. Because it's multiple changes, adding what they are in comments is a bit more difficult that just making the changes and making a PR to your branch. That way if you accept them into your branch, I just have to re-pull from your branch to local without worrying about merge conflicts.

File: tests/integrations/wscfileexporter/test_plan.sh [hunk 1]
```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+echo `pwd`
+
+SERVICE_NAME="water-auth-server"
```
Comment (isuftin) [inline tests/integrations/wscfileexporter/test_plan.sh]: The `SERVICE_NAME` variable here appears in each of these test plans. I don't think it's being used anywhere and should probably be removed from all of these test_plan shell scripts

File: tests/integrations/wscfileexporter/test_plan.sh [hunk 2]
```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+echo `pwd`
```
Comment (isuftin) [inline tests/integrations/wscfileexporter/test_plan.sh]: Is this necessary throughout?

**Comment by isuftin** [comment]
This looks good. I'm able to run each of the tests locally without fail and it seems like all of the jmeter tests do pass.

**Comment by zmoore-usgs** [comment]
Seems reasonable to me. I can add that real quick

**Comment by isuftin** [comment]
Maybe create a JIRA for that because it would also tie in to a new Jenkins job to run that or an update to the current job to be able to run that

**Comment by zmoore-usgs** [comment]
Seems reasonable
